seismostats/analysis/tests/test_estimate_mc.py

```diff
@@ -1,11 +1,40 @@
 import pickle
-
 import numpy as np
 import pytest
+import pandas as pd
 from numpy.testing import assert_allclose, assert_almost_equal, assert_equal
 
-from seismostats.analysis.estimate_mc import (empirical_cdf, mc_ks,
-                                              mc_max_curvature)
+from seismostats.io.client import FDSNWSEventClient
+from seismostats.analysis.estimate_mc import (empirical_cdf,
+                                              mc_ks,
+                                              mc_max_curvature,
+                                              mc_by_bvalue_stability)
+
+
+@pytest.fixture
+def swiss_2023_magnitudes():
```

Comment:
I don't like that we are dependent that the server is always working for the testing. I We should instead have a static catalog (I guess a csv?)

Reply:
To be discussed with Nicolas at our next meeting.

Reply:
Took care of it :)